### ISSUE TITLE
cli: pretty-printed key parsing a bit flaky

### ISSUE THREAD

**Issue by mberhault** [cockroachdb/cockroach#5671]
eg:
```
./cockroach debug keys --from="/Min" --to='/System/"tsd\x12cr.node.txn.durations-1m-p50\x00\x01\x89\xf8\x06/a2"' .
Error: unsupported pretty key: known key, but unsupported subtype
```

I assume it referes to `/System/....`

When trying a shortened version of the `--to` key:
```
./cockroach debug keys --from="/Min" --to=/System .
Error: constructed key deviates from original: /Min vs /System
```

I have no idea what that even means.

**Comment by tschottdorf** [comment]
pretty-printed key parsing is incomplete and experimental. If you need that
key parsed, you can add the implementation and/or improve the error. In
this particular case, I think the message is telling you that /System
erroneously comes back as /Min.

**Comment by tamird** [comment]
This implementation has had some recent attention from @tschottdorf in #7427. Hopefully this has been addressed in that commit; either way, this doesn't seem actionable, so I'm going to close.

**Comment by tschottdorf** [comment]
I think we could even phase out that functionality. I haven't found it
useful recently. But this issue can stay closed.

On Thu, Jun 30, 2016, 19:34 Tamir Duberstein <notifications@github.com>
wrote:

> Closed #5671 <https://github.com/cockroachdb/cockroach/issues/5671>.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/cockroachdb/cockroach/issues/5671#event-709883903>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AE135DYv4xLXYtIrLkNSplnm27NMebXjks5qRFKCgaJpZM4H61yG>
> .
>
-- 

-- Tobias
